### ISSUE TITLE
Cooking and Cakes

### ISSUE THREAD

**Issue by SilverTalismen** [PolarisSS13/Polaris#3980]
#### Brief description of the issue
Cake ingredients once again react within microwaves meaning you can't make the recipe cakes and likely other recipies
#### What you expected to happen
to be able to make a cake
#### What actually happened
It reacted with the eggs making dough while IN the microwave/
#### Steps to reproduce
add flour
add suger
add eggs- BAM dough!
#### Additional info:
 - **Server Revision**:master - 2017-09-28
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

**Comment by Anewbe** [comment]
Eggless cakes Soon™️

**Comment by Wutnold124** [comment]
vegan cakes are horrible

**Comment by Anewbe** [comment]
You don't need eggs for cake IRL, that's a lie by Big Dairy.

**Comment by SilverTalismen** [comment]
Ok but seriously this getting fixed would be nice.

**Comment by Anewbe** [comment]
Yeah, I'm serious about the eggless cakes, might be able to have the PR up within the evening.

**Comment by Anewbe** [comment]
Did some testing: It's still possible to make cake, but you need to add egg **yolk** to the microwave, instead of the whole egg.

**Comment by Iamgoofball** [comment]
>You don't need eggs for cake IRL, that's a lie by Big Dairy.

are you high

**Comment by Anewbe** [comment]
Not recently, no.
